e2e/cypress/integration/emoji/react_to_last_message_shortcut_spec.js

```diff
@@ -42,14 +42,6 @@ describe('Keyboard shortcut for adding reactions to last message in channel or t
         });
     });
 
```

Comment:
Getting this failure consistently
![Screen Shot 2020-03-19 at 10 33 04 AM](https://user-images.githubusercontent.com/487991/77097867-b0496180-69ce-11ea-986d-b40030652541.png)

Reply:
Oh I see.  It's not related to this PR but a bug introduced by https://github.com/mattermost/mattermost-webapp/commit/c6c1b5e205ac1994e3c41247fdb687e6b3ac4774.  I suggests to deal with it separately.